lib/runner.js

```diff
@@ -98,10 +105,10 @@ class Runner {
                 }, 0);
 
                 if (warnings > maxWarnings) {
-                    return reject({
-                        message: '',
-                        status: EXIT_CODES.WARNING
-                    });
+                    return reject(new RunnerError(
+                        '',
```

Comment:
I don't wanna unnecessary output in case someone's piping the output somewhere else, reporters will already have printed everything that's needed.